.github/workflows/node.js-tests-upcoming.yml

```diff
@@ -1,4 +1,4 @@
-name: Node.js CI (Next)
+name: Node.js CI - Test Upcomming Changes
```

Comment:
```suggestion
name: Node.js CI - Test Upcoming Changes
```

Reply:
I caught that in a self-review, and added a commit.